docs/gitrepo-structure.md

```diff
@@ -48,6 +48,9 @@ kustomize:
   # Use a custom folder for kustomize resources. This folder must contain
   # a kustomization.yaml file.
   dir: ./kustomize
+  # Specify build options for kustomize build.
+  # Default: ""
+  buildOptions: --load_restrictor none
```

Comment:
Since `buildOptions` essentially captures `kustomize build` command-line arguments and since both ArgoCD and FluxCD capture "build options" as a simple string, I was thinking Fleet would follow suite and use a simple string as well. Introducing a list here will likely make validation more complex. Currently, the patch simply splits the `buildOptions` on a space delimiter (here)[https://github.com/ron1/fleet/blob/e6043427495b6b174c44bfadd3e7f683224cc441/pkg/kustomize/kustomize.go#L98] and relies on the kustomize `build.Validate` for validation.

Reply:
That makes sense. If the pattern has been set using a space delimiter, this is a good idea.